server/index.js

```diff
@@ -4,16 +4,30 @@ const path = require('path');
 const app = express();
 const port = 3000;
 
-console.log(path.join(__dirname, '..', '/client/dist'));
-
 app.use(express.static(path.join(__dirname, '..', '/client/dist')));
 app.use(express.json());
 app.use(express.urlencoded({extended: true}));
 
-// app.get('/', function(req, res) {
-//   res.status(200);
-//   res.end();
-// })
+// ========== Shared ========== //
+
+// ========== Overview ========== //
+
+// ========== Related Products ========== //
+
+// ========== Questions & Answers ========== //
+
+app.get('/qa/questions/:product_id', function(req, res) {
+  const apiCalls = require('../utils/apiCalls.js');
```

Comment:
Just a suggestion, but maybe we could put that require on line 20 outside of the scope of that function so we all can pull from the same variable as apposed to re-declaring the variable.

Reply:
Good call, moved the declaration to top of doc